README.md
# Afflicted

![Снимок экрана 2022-06-09 090726](https://user-images.githubusercontent.com/57856647/172793463-7f02be76-306d-4cc0-9f6c-3c4f3dd3941a.jpg)

## Changes

* cooldown text to [Cooldown](https://wowpedia.fandom.com/wiki/API_Cooldown_SetCooldown)
* Added display of the following abilities:
  * [Hex](https://base.opiums.eu/?spell=51514)
  * [Howl of Terror](https://base.opiums.eu/?spell=17928)
* Added `xOffset` and `yOffset` sliders to fine-tune the position  

***

*If you previously had an addon installed and you replaced it with this one, and for some reason the addon stopped working for you, then delete the files `Affected.lua` and `Affected.lua.bak` along the path `WTF\Account\name\SavedVariables`*


changelog.txt
commit fafcff3f22d9f01adaa0d3b06576538fa4db5ef7
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Sat Feb 6 10:06:14 2010 -0800

    Library update

commit cd9529998e8f2a28fcb525b56c79c0c9164e80fa
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Sat Feb 6 10:04:00 2010 -0800

    Fixed index nil error at Afflicted.lua:79, probably shouldn't set a table with a known nil index!

commit d596100f66e590b929a92d93bd996a2796d71f44
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Tue Dec 22 01:21:15 2009 -0800

    Bug fix when accessing Afflicted.spells with a nil index

commit 11b511b05a7e74da35effa0ce90ff7d6c434d771
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Sat Dec 5 09:31:14 2009 -0800

    - Reapplied the #120 fix

commit 6f18b513c949ebe05e3614f166199f01a5d94733
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Fri Dec 4 17:18:34 2009 -0800

    Revert "- Fixed error on line 120"
    
    This reverts commit c12a2ffe1b1ac33ee6ca2171a2129dcea2d0a105.

commit c12a2ffe1b1ac33ee6ca2171a2129dcea2d0a105
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Fri Dec 4 17:17:45 2009 -0800

    - Fixed error on line 120

commit 0bf30a7c93950d6d88bc62786441906359cfd403
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Fri Dec 4 09:05:00 2009 -0800

    Divine Sacrifice, 10 second duration -> 6 seconds
    Aura Mastery, 10 second duration -> 6 seconds

commit 0c793238cfc931b7a61f4efc4d9df8a667043e71
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Fri Dec 4 09:04:18 2009 -0800

    Removed unneeded library files

commit 0dd691b4c7b83eede50a7d2020212d344bb931ce
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Fri Dec 4 08:53:08 2009 -0800

    Tweaked default loading of spells

commit 43198a6ba08a108e694664dc77c1574556cbe46a
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Fri Dec 4 08:52:58 2009 -0800

    TOC Bump 30300

commit da495ff9e0b1d2dc4be28788b297e6ff03d70e4d
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Fri Dec 4 08:44:30 2009 -0800

    - Library update

commit 07d12ef8c5dc611677a912879fde8172649fb5ce
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 9 16:31:37 2009 -0700

    - Changed spell updating, only duration, cooldown, type and class will be merged meaning anchor, enabled/disabled status and so on will not be reset when updating spells

commit fff115f9f0e86e53eca5d64b49fa82bf2b5dd377
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 9 16:30:52 2009 -0700

    - Bestial Wrath, 18 seconds -> 10
    - Vampiric Blood, 120 second cooldown -> 60 second cooldown, 20 second duration -> 10 econd duration

commit 2fb397fd81fe76ab4f1393e0db391b8375475489
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Thu Sep 3 19:05:59 2009 -0700

    - Library update

commit 17b361923ff831ed9845b53ba02f919a77e9eeb4
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Thu Sep 3 19:05:24 2009 -0700

    - Localization update

commit 4dac08361899280c42d7755349e6decef57556fa
Merge: 0830174 2af997b
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 2 18:40:23 2009 -0700

    Merge branch 'master' of github.com:Shadowed/Afflicted

commit 0830174bb1aaa1c6b8db810d4f8461e5e35077a4
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 2 18:39:09 2009 -0700

    - Scale can no longer be set to 0
    - Added ConfigMode

commit ce864b8f0986414270db4e53a4d92051a3df85c8
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 2 18:38:30 2009 -0700

    - I am stupid and selected the wrong file, bars and icons will now be staggered in their position, first one is at 0, 0 second one is at 0, 20 and so on so they aren't just a cluster of anchors to start with

commit d265ab9d39ff0a8d56c204127d0857e6a5e48c0c
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 2 18:37:27 2009 -0700

    - Localization update

commit 4bc4ff094eda8f181dd4ce2ee0a87d703c23b8a6
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 2 18:36:58 2009 -0700

    - Anchors are now unlocked by default, added a new help frame (similiar to Shadowed Unit Frames) that indicates where you go to lock them as well as a button to lock them

commit 64e68c7ad128c9e0a5e617a2ab070951a7de2ffe
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 2 18:15:06 2009 -0700

    - Fixed icons requiring ALT + Drag while bars only required dragging

commit fcbdbbefb2545e489c4c7b6eb8fdc792279f6068
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Wed Sep 2 18:14:52 2009 -0700

    - Fixed error in bars.lua

commit 2af997b7a048636c800e1925b6b76d8683566289
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Thu Aug 27 21:53:49 2009 -0700

    - Fixed error in bars.lua:134

commit 2661f136555a6990143d78271c2c735fac232b58
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Tue Aug 11 09:40:46 2009 -0700

    - Fixed an attempt to index field GTB a nil value error

commit de333ecd25bf0617e8f21997a19d9648ffbcefb3
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Mon Aug 10 12:45:27 2009 -0700

    - Updated to support the configName/cooldownName/name fields in spells so I can customize stuff a bit more

commit 505ed037981aa9fe4e8a4cac4dca7b7f60fee548
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Mon Aug 10 12:44:09 2009 -0700

    - Cleaned up bar code a little bit

commit b11f418c73deaf8551e1e34c505f54a6f6ebcb71
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Mon Aug 10 12:43:45 2009 -0700

    - Changed to support the config name in spells

commit 67577c42a063e2ef963c3cbc6deebe1092a74494
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Mon Aug 10 12:42:09 2009 -0700

    - Library update

commit 7c3c9d3a246a23498b2b18386ccee5ab00c5b95a
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Mon Aug 10 12:06:26 2009 -0700

    - Library update

commit 7319799545694086b9e19608f2d29776c228586a
Author: Shadowed <shadowed.wow@gmail.com>
Date:   Mon Aug 10 12:06:02 2009 -0700

    - Added lower ranks of Psychic Scream

commit 368df095a4fe8670f01d89ee8a3809c1f7d9c1eb
Author: Shadowed <shadowed.wow@gmail.com>

spells.lua
---@class AfflictedSpells
AfflictedSpells =
{
    version = 5;
    spells = {
        -- Death Knight
        -- Strangulate
        [47476] = "{cooldown=120;cdAnchor='interrupts';class='DEATHKNIGHT';}",
        -- Icebound Fortitude
        [48792] = "{disabled=true;duration=12;cooldown=120;cdDisabled=true;anchor='defenses';cdAnchor='cooldowns';class='DEATHKNIGHT';}",
        -- Mind Freeze
        [47528] = "{cooldown=10;cdAnchor='interrupts';class='DEATHKNIGHT';}",
        -- Anti-Magic Shell
        [48707] = "{cdDisabled=true;type='buff';disabled=true;duration=5;cooldown=45;anchor='defenses';cdAnchor='cooldowns';class='DEATHKNIGHT';}",
        -- Death Grip
        [49576] = "{cdDisabled=true;cooldown=35;cdAnchor='spells';class='DEATHKNIGHT';}",
        -- Anti-Magic Zone
        [51052] = "{disabled=true;duration=10;anchor='defenses';cooldown=120;cdDisabled=true;cdAnchor='cooldowns';class='DEATHKNIGHT';}",
        -- Dancing Rune Weapon
        [49028] = "{disabled=true;duration=17;anchor='damage';cooldown=90;cdDisabled=true;cdAnchor='cooldowns';class='DEATHKNIGHT';}",
        -- Lichborne
        [49039] = "{duration=10;anchor='defenses';cooldown=120;cdDisabled=true;cdAnchor='cooldowns';class='DEATHKNIGHT';}",
        -- Vampiric Blood
        [55233] = "{duration=10;anchor='defenses';cooldown=60;cdDisabled=true;cdAnchor='cooldowns';class='DEATHKNIGHT';}",
        -- Leap (Ghoul)
        [47482] = "{cooldown=20;cdDisabled=true;cdAnchor='spells';class='DEATHKNIGHT';}",
        -- Gnaw (Ghoul)
        [47481] = "{cooldown=30;cdDisabled=true;cdAnchor='spells';class='DEATHKNIGHT';}",

        -- Paladin
        -- Ardent Defender
        [66233] = "{type='buff';disabled=true;cooldown=120;cdAnchor='cooldowns';class='PALADIN';}",
        -- Divine Plea
        [54428] = "{type='buff';disabled=true;duration=15;anchor='buffs';cooldown=60;cdDisabled=true;cdAnchor='cooldowns';class='PALADIN';}",
        -- Avenging Wrath
        [31884] = "{duration=20;anchor='damage';cooldown=120;cdDisabled=true;cdAnchor='cooldowns';class='PALADIN';}",
        -- Hammer of Justice
        [10308] = "{cooldown=40;cdDisabled=true;cdAnchor='cooldowns';class='PALADIN';}",
        -- Hand of Protection
        [10278] = "{type='buff';duration=10;anchor='defenses';cdDisabled=true;cooldown=180;cdAnchor='cooldowns';class='PALADIN';}",
        -- Divine Shield
        [642] = "{type='buff';duration=12;anchor='defenses';cooldown=300;cdDisabled=true;cdAnchor='cooldowns';class='PALADIN';}",
        -- Hand of Freedom
        [1044] = "{type='buff';duration=14;anchor='spells';cdDisabled=true;cooldown=25;cdAnchor='cooldowns';class='PALADIN';}",
        -- Hand of Sacrifice
        [6940] = "{disabled=true;type='buff';duration=12;anchor='defenses';cdDisabled=true;cooldown=120;cdAnchor='cooldowns';class='PALADIN';}",
        -- Aura Mastery
        [31821] = "{duration=6;anchor='defenses';cooldown=120;cdAnchor='cooldowns';class='PALADIN';}",
        -- Divine Sacrifice
        [64205] = "{type='buff';duration=6;anchor='defenses';cooldown=120;cdDisabled=true;cdAnchor='cooldowns';class='PALADIN';}",

        -- Warrior
        -- Shield Reflect
        [23920] = "{type='buff';disabled=true;duration=5;cdDisabled=true;cooldown=10;cdAnchor='buffs';class='WARRIOR';}",
        -- Shield Bash
        [72] = "{cooldown=12;cdAnchor='interrupts';class='WARRIOR';}",
        -- Pummel
        [6552] = "{cooldown=10;cdAnchor='interrupts';class='WARRIOR';}",
        -- Intervene
        [3411] = "{cooldown=30;cdDisabled=true;cdAnchor='defenses';class='WARRIOR';}",
        -- Recklessness
        [1719] = "{duration=12;anchor='damage';cooldown=300;cdDisabled=true;cdAnchor='cooldowns';class='WARRIOR';}",
        -- Charge
        [11578] = "{cooldown=13;cdAnchor='spells';class='WARRIOR';}",
        -- Berserker Rage
        [18499] = "{disabled=true;duration=10;anchor='spells';cooldown=30;cdAnchor='cooldowns';class='WARRIOR';}",
        -- Intercept
        [20252] = "{cooldown=15;cdAnchor='spells';class='WARRIOR';}",
        -- Shield Wall
        [871] = "{disabled=true;duration=12;anchor='defenses';cooldown=300;cdDisabled=true;cdAnchor='cooldowns';class='WARRIOR';}",
        -- Disarm
        [676] = "{cdDisabled=true;cooldown=60;cdAnchor='spells';class='WARRIOR';}",

        -- Druid
        -- Survival Instincts
        [61336] = "{duration=20;anchor='defenses';cooldown=300;cdDisabled=true;cdAnchor='cooldowns';class='DRUID';}",
        -- Berserk
        [50334] = "{duration=15;anchor='damage';cooldown=180;cdDisabled=true;cdAnchor='cooldowns';class='DRUID';}",
        -- Nature's Grasp
        [17329] = 53312,
        [27009] = 53312,
        [53312] = "{type='buff';disabled=true;duration=45;anchor='buffs';cooldown=60;cdDisabled=true;cdAnchor='cooldowns';class='DRUID';}",
        -- Innervate
        [29166] = "{type='buff';disabled=true;duration=10;anchor='buffs';cooldown=180;cdDisabled=true;cdAnchor='cooldowns';class='DRUID';}",
        -- Frenzied Regeneration
        [22842] = "{duration=10;anchor='defenses';cooldown=180;cdDisabled=true;cdAnchor='cooldowns';class='DRUID';}",
        -- Feral Charge - Bear
        [16979] = "{cooldown=15;cooldownName='feral';cdAnchor='interrupts';class='DRUID';}",
        -- Feral Charge - Cat
        [49376] = "{cooldown=30;cooldownName='feral';cdAnchor='spells';class='DRUID';}",
        -- Bash
        [8983] = "{cooldown=30;cdDisabled=true;cdAnchor='interrupts';class='DRUID';}",

        -- Priest
        -- Hymn of Hope
        [64901] = "{duration=8;anchor='buffs';cooldown=360;cdDisabled=true;cdAnchor='cooldowns';class='PRIEST';}",
        -- Dispersion
        [47585] = "{duration=6;anchor='defenses';cooldown=75;cdAnchor='cooldowns';class='PRIEST';}",
        -- Guardian Spirit
        [47788] = "{type='buff';duration=10;anchor='defenses';cooldown=180;cdDisabled=true;cdAnchor='cooldowns';class='PRIEST';}",
        -- Pain Suppression
        [33206] = "{type='buff';duration=8;anchor='defenses';cooldown=144;cdDisabled=true;cdAnchor='cooldowns';class='PRIEST';}",
        -- Silence
        [15487] = "{cooldown=45;cdAnchor='interrupts';class='PRIEST';}",
        -- Psychic Scream
        [8122] = 10890,
        [8124] = 10890,
        [10888] = 10890,
        [10890] = "{cooldown=26;cdAnchor='spells';cdDisabled=true;class='PRIEST';}",

        -- Warlock
        -- Fel Domination
        [18708] = "{type='buff';duration=15;anchor='spells';cooldown=180;cdDisabled=true;cdAnchor='cooldowns';class='WARLOCK';}",
        -- Devour Magic (Felhunter)
        [27276] = 48011,
        [27277] = 48011,
        [48011] = "{cooldown=8;cdAnchor='spells';cdDisabled=true;class='WARLOCK';}",
        -- Intercept (Felguard)
        [30198] = 47996,
        [47996] = "{cooldown=30;cdDisabled=true;cdAnchor='spells';cdDisabled=true;class='WARLOCK';}",
        -- Spell Lock (Felhunter)
        [19647] = "{cooldown=24;cdAnchor='interrupts';class='WARLOCK';}",

        -- Shaman
        -- Hex
        [51514] = "{cooldown=45;cdDisabled=true;cdAnchor='spells';class='SHAMAN'}",
        -- Wind Shock
        [57994] = "{cooldown=6;cdAnchor='interrupts';class='SHAMAN';}",
        -- Heroism
        [32182] = 2825,
        -- Bloodlust
        [2825] = "{type='buff';duration=40;anchor='damage';cooldown=300;cdDisabled=true;cdAnchor='cooldowns';class='SHAMAN';}",
        -- Tremor Totem
        [8143] = "{type='totem';disabled=true;duration=5;anchor='buffs';repeating=true;class='SHAMAN';}",
        -- Shamanistic Rage
        [30823] = "{disabled=true;duration=15;anchor='defenses';cooldown=60;cdDisabled=true;cdAnchor='cooldowns';class='SHAMAN';}",
        -- Mana Tide Totem
        [16190] = "{type='totem';disabled=true;duration=12;anchor='buffs';cooldown=300;cdAnchor='cooldowns';class='SHAMAN';}",
        -- Grounding Totem
        [8177] = "{type='totem';disabled=true;duration=45;anchor='buffs';cooldown=15;cdAnchor='cooldowns';class='SHAMAN';}",

        -- Hunter
        -- Bestial Wrath
        [19574] = "{duration=10;anchor='damage';cdDisabled=true;cooldown=120;cdAnchor='cooldowns';class='HUNTER';}",
        -- Wyvern Sting
        [27068] = 49012,
        [49011] = 49012,
        [49012] = "{cooldown=60;cdAnchor='cooldowns';class='HUNTER';}",
        -- Silencing Shot
        [34490] = "{cooldown=20;cdAnchor='interrupts';class='HUNTER';}",
        -- Readiness
        [23989] = "{cooldown=180;cdDisabled=true;cdAnchor='cooldowns';resets={49012,34600,63670,19263,3034,14327,34490};class='HUNTER';}",
        -- Nether Shock (Nether Ray)
        [53588] = 53589,
        [53589] = "{cooldown=40;cdAnchor='interrupts';class='HUNTER';}",
        -- Pin (Crab)
        [53547] = 53548,
        [53548] = "{cooldown=40;cdDisabled=true;cdAnchor='spells';class='HUNTER';}",
        -- Pummel (Gorilla)
        [26090] = "{cooldown=30;cdAnchor='interrupts';class='HUNTER';}",
        -- Frost Trap
        [13809] = 14311,
        -- Freezing Arrow
        [60210] = 14311,
        -- Freezing Trap
        [14311] = "{type='trap';configName='frost';cooldownName='frost';disabled=true;duration=30;anchor='spells';cooldown=30;cdDisabled=true;cdAnchor='cooldowns';class='HUNTER';}",
        -- Explosive Trap
        [27025] = 49056,
        [49066] = 49056,
        [49067] = 49056,
        -- Black Arrow
        [63670] = 49056,
        [63671] = 49056,
        [63672] = 49056,
        -- Immolation Trap
        [27023] = 49056,
        [49055] = 49056,
        [49056] = "{type='trap';configName='fire';cooldownName='fire';disabled=true;duration=30;anchor='spells';cooldown=30;cdDisabled=true;cdAnchor='cooldowns';class='HUNTER';}",
        -- Snake Trap
        [34600] = "{type='trap';configName='nature';cooldownName='nature';disabled=true;duration=30;anchor='spells';cooldown=30;cdDisabled=true;cdAnchor='cooldowns';class='HUNTER';}",
        -- Deterrence
        [19263] = "{duration=5;anchor='defenses';cooldown=90;cdDisabled=true;cdAnchor='cooldowns';class='HUNTER';}",
        -- Viper Sting
        [3034] = "{cooldown=15;cdAnchor='spells';class='HUNTER';}",

        -- Mage
        -- Cold Snap
        [11958] = "{cooldown=384;cdDisabled=true;cdAnchor='cooldowns';resets={12472,44572,45438};class='MAGE'}",
        -- Deep Freeze
        [44572] = "{cooldown=30;cdAnchor='spells';class='MAGE';}",
        -- Icy Veins
        [12472] = "{type='buff';duration=20;anchor='damage';cooldown=144;cdDisabled=true;cdAnchor='cooldowns';class='MAGE';}",
        -- Invisibility
        [66] = "{disabled=true;duration=23;anchor='defenses';cooldown=180;cdDisabled=true;cdAnchor='cooldowns';class='MAGE';}",
        -- Ice Block
        [45438] = "{type='buff';duration=10;anchor='defenses';cooldown=240;cdDisabled=true;cdAnchor='cooldowns';class='MAGE';}",
        -- Counterspell
        [2139] = "{cooldown=24;cdAnchor='interrupts';class='MAGE';}",
        -- Blink
        [1953] = "{cooldown=15;cdDisabled=true;cdAnchor='cooldowns';class='MAGE';}",

        -- Rogue
        -- Kick
        [1766] = "{cooldown=10;cdAnchor='interrupts';class='ROGUE';}",
        -- Kidney Shot
        [8643] = "{cooldown=20;cdAnchor='spells';class='ROGUE';}",
        --[51713] = "{duration=10;anchor='damage';cooldown=120;cdDisabled=true;cdAnchor='cooldowns';class='ROGUE';}",
        -- Shadow Step
        [36554] = "{disabled=true;duration=3;anchor='buffs';cooldown=30;cdAnchor='cooldowns';class='ROGUE';}",
        -- Adrenaline Rush
        [13750] = "{duration=15;anchor='damage';cooldown=180;cdDisabled=true;cdAnchor='cooldowns';class='ROGUE';}",
        -- Preparation
        [14185] = "{cooldown=300;cdDisabled=true;cdAnchor='cooldowns';resets={14177,36554,26889,11305,26669};class='ROGUE'}",
        -- Cloak of Shadows
        [31224] = "{duration=5;anchor='defenses';cooldown=60;cdDisabled=true;cdAnchor='cooldowns';class='ROGUE';}",
        -- Vanish
        [26889] = "{type='buff';disabled=true;duration=10;anchor='defenses';cooldown=120;cdAnchor='cooldowns';class='ROGUE';}",
        -- Sprint
        [11305] = "{duration=15;disabled=true;anchor='buffs';cooldown=120;cdAnchor='cooldowns';class='ROGUE';}",
        -- Evasion
        [26669] = "{duration=15;anchor='defenses';cooldown=120;cdDisabled=true;cdAnchor='cooldowns';class='ROGUE';}",
        -- Blind
        [2094] = "{cooldown=120;cdAnchor='cooldowns';class='ROGUE';}",
        -- Dismantle
        [51722] = "{cooldown=60;cdDisabled=true;cdAnchor='cooldowns';class='ROGUE';}",

        -- Misc
        -- PvP Trinket
        [59752] = 42292,
        [42292] = "{cooldown=120;cdAnchor='defenses'}",
        -- Escape Artist
        [20589] = "{cooldown=105;cdAnchor='defenses'}",
    };

    totems = {
        [GetSpellInfo(8227)] = "fire",
        [GetSpellInfo(8181)] = "fire",
        [GetSpellInfo(2894)] = "fire",
        [GetSpellInfo(8499)] = "fire",
        [GetSpellInfo(10585)] = "fire",
        [GetSpellInfo(6363)] = "fire",
        [GetSpellInfo(57720)] = "fire",
        [GetSpellInfo(8170)] = "water",
        [GetSpellInfo(8184)] = "water",
        [GetSpellInfo(5394)] = "water",
        [GetSpellInfo(5675)] = "water",
        [GetSpellInfo(16190)] = "water",
        [GetSpellInfo(8143)] = "earth",
        [GetSpellInfo(2062)] = "earth",
        [GetSpellInfo(8071)] = "earth",
        [GetSpellInfo(8075)] = "earth",
        [GetSpellInfo(2484)] = "earth",
        [GetSpellInfo(5730)] = "earth",
        [GetSpellInfo(8177)] = "air",
        [GetSpellInfo(10595)] = "air",
        [GetSpellInfo(6495)] = "air",
        [GetSpellInfo(8512)] = "air",
        [GetSpellInfo(3738)] = "air",
    }
}


function AfflictedSpells:GetData()
    return self.spells;
end

function AfflictedSpells:GetTotemClass(spellName)
    return self.totems[spellName];
end
